python/examples/DFtoVW_tutorial.ipynb

```diff
@@ -0,0 +1,1521 @@
xgOFFjAsFDFKJqXh0azleJRyfVFhtWbdRK6rjqMicFCm2C5wNrc8e1TSWsflquNkoqkrkEZwmKY8uN7ywmFJbR1PF1smecJia8m0YJgZUgOvnS+52Fq6GKja/nUwLhLefzriq+/t8eKtIa+crdl2Hds2MkxAKsEoTzhftjyab9BS8sLxiL1BwsW64eWzNU8XNTY4Bjqy3DYQBTbtJ96dLRqSRPLiaMBLR0OyRHOaaKa54cF1xTBR1C4iAzSd58HllsOyH1hwtWmZ3UzCvD0tWLeeuvPkRr3L4eRCeNfkunnVMUz7nrqdAPVFzb8NfMPN538R+Cf0ItO/Dfz3McYI/HMhxEQIcRpjfPo52cp/RXal3z87Avgd3/F9u9jcjh07duz4rNmJTDu+qPmZpitpJdFK9l0nRpMbxTP7/aJrWfcOp/ccl4go+OCzY2ZbzyjX+BD48P05L5+v2FqHSiX7WWRRd1yvW1aNJVOKbecgRoyJ1F3HDz2Y8YnLJVebFi0lwTvuTTNCjMzrhgczT20dZ8uGo2HGYttw7yDnqEgwEq42nr1S4aPj9cuOaS5w3rOXGzZtxeVGUbUtAkEiYGAUT2dbTqc5i6Xlvm155kCzaTx7uWbmHXuJ5vGyQyvPvWnGk3WNkYIuRlwItB2IALkUCCV5vOm4UxouWw/OsTdMWLcBb2HTBvJUMcgMmzriteCiduRKUGSGPSlYoXBdoNESayNlKhimgoX1TIf9VDUNtN5zZiOZlhRakMheHApC4G/idPOmY96B6yLHo5TOBbY+cJAapBRcVx0HhWFWO0yuWdq+WDuRoJTExMgoM1xbj5L9NkqgDREjwfpI5QJaS9ouIDTMG4eLkUFmiMBl7ZCJZM8YBJAIQa087zHnBHNBUEd8fHtMnvdC1QflP+F3Tv8y++qab5n9Rd6ocxQwSDW/afTXeF/ycX60+Wp+0P1qsJ4qeMjejqqp7oJF4xBSsAE+trnFS/oDXPk9EtX37awCROf5o9d/mpUYMfNjOiXwPpAryRP3DP8Dv4ehE5zVHVL1jq1BaVhXHRjFKFFctr4vQ/eRvOgn5bUCUutxEpqN5WCQkCVwXQdWMiIcJKnGIJhXfcn5vXHKa4uaZyc5m6bjaJBwtmkZJxpjVN8PpgWpkzjryEvDrPacjhNwkCRw2XjatedkkjDqYFRmnC22FIlia+FsZTnKE0ajhNcuKrZ1IE8FVW0ZJ5rpGHQj8EHhgE3bkRjNKE15Om8wShOCoNnAw7XlSwdDDgcKyCkymG07ou9FSwFcrxyLakPtO0KUpDoSnWC2afmh164Z5AlaK2yIvHq24WNP1jjn2CsNqY7U0aNQrDvPbNEwSnsR7pMXG8aZofOB81VN7TqM6aN7USm2ViCl4Hics60dOkoWdcdJomms48cfLhBCsqhWpEaRKIntAq9fVRwOM/YGKS+fb4gxcjLK3lH6Xbx1Tnzztr4wvHnrtXe2bJipvl8Lfnqf3Y4vSCLwD4QQEfgLMcbvAI7fIRydAcc3n98GHr7jex/d3PYFJTLtonKfOTrJPtebsGPHjh07voDZiUw7vqh5Z/cI8NNKv/tF1duLrCzRfPJixRuLmtSUSKl4/aqm7fpR9RBZ1x3rti/SNYDz/cjyO9OMZeMxEnKleOVyg1aRpgtcrhq0Srk7LpnVFfMaxg72y5yBkQgkpRZsrScxkcObCWtrF3gwrzhfdXzo2SFCRD7x5IoOxeNFw+0JHI8zltuGysK2ddyZpigZCbVg01rG44TlvKZr+2luQgQOR5rcCG6h2FYdi1xju/75JUJQ+8CqCmx8xPi+TDsjUkWILnI4zLCu5XismW09WgkutpZpqlnZjoMiJQPmwKayFEXCJy4rvvSw4GJtORomrBrLUZYw31huJ4ptiFgXuVVovOn7j5wLWN+XXC8bx3iQcH/RcG+S0bmALhSXlaPMFK2DIlWsq45x0YtGxgjmW8s0N2gX+nLqTUM5yLhoOpoAaWGorMdryco6nk9SfCbJJDxZN0yHKVMCGwu50aQCvkR9P1+5/2P44Pnrze/n1WXD3iDFOsefPPr3kCLyXZt/j1fD72Kx7cgTyX4ZecF8EgBVP6HM3kcp+ljdPOwDUKiGpnMY0wuaT+yU/3rxu1n6PR7Lr6BIFG0XSIH/yX8T//j83wEhMFqy9RHnPKeF4b57jtW649Yo4VHVQapoWw8xkKWaJ61jkBtmG0uQ0FSeUWmoGk+ZS+ZtZD/RXFhH6gQ6VYQQcdYzSjR50XdenWIoMk0WIkkm+94tKXAusBWCPS24N8mZVR3TQcLWevYHKbhAB2xdYD8xQORoYBjmUOY5ZQrLxrJYBo4TRZcorjcOpQRSKgSS7drTRRhnCdKAlI6jkeZi5RiUCbaLNM6ztYoyTVhVFqlgkis6FxHBkWWap7Oak2lKaSI6TUl1ZJLnFGnGqmlZNy3jXLOsOga5wdNhZGDbeL72mT2uqoYffTAjkZKN75hIw1ffHXFrWvDgaouWkdYKXp9V3N0reXZa0oXA19xLWdcOGwUPrrdsW8+yavmqexNePCk5nzc8XbZcbhue38upusi0MNTWMW86Yny7xPvBdcXDec040+wN0rdE81GuWdWOZ/YLtJK853jQx+cG6VtupDfPhZ8qyEd6sWmSGxhn73Iy7fii4OtjjI+FEEfAPxRCfPydd8YY440A9RkjhPjdwO8GuHfv3s/flv48sYvK7dixY8eOHb8w7ESmHV/U/LSibwmtczj/tpvpnVflnQ8cDlM+eGfMV9wbsawCSSL4yBsLPnm54dYk5d5hyek04+mi5WLd8LEnS0KEr767x7jQBNdH08pUkUpNFVpePM04yHI8gSOdMs1TfBQMU8n5tiVERyoNdycZo9Qwa2q6AJumpUgiw1Qzr1qOBjnvPy3RyhKvNUUimeaK2mrGA8nTeUuRBgqTkCqBFIJ1Zbk1zsi0p/aOItNUreeVWcfx0BBkxAcoU4NtIxvryI3EIDkpIHpY
```

Comment:
nit: maybe link to simple label in wiki [here](https://github.com/VowpalWabbit/vowpal_wabbit/wiki/Input-format#simple)?

Reply:
You meant in the place where I introduce `SimpleLabel` ? If yes I've just added a link in my last commit 6115e12 .

Line 1015 to 1018 seems to be before that but I might have misunderstood your comment.

Reply:
looks good